docs/topics/spider-middleware.rst

```diff
@@ -328,6 +328,68 @@ Default: ``True``
 
 Whether to enable referer middleware.
 
+.. setting:: REFERER_POLICY
+
+REFERER_POLICY
+^^^^^^^^^^^^^^
+
+.. versionadded:: 1.4
+
+Default: ``'scrapy.spidermiddlewares.referer.DefaultReferrerPolicy'``
+
+.. reqmeta:: referrer_policy
+
+`Referrer Policy`_ to apply when populating Request "Referer" header.
+
+This setting accepts:
+
+- a path to a ``scrapy.spidermiddlewares.referer.ReferrerPolicy`` subclass,
+  either a custom one or one of the built-in ones
+  (see ``scrapy.spidermiddlewares.referer``),
```

Comment:
What kind of reference do you have in mind? built-in classes?

We could also define an interface for policies, so people know what to implement.